src/System.IO.FileSystem/tests/Directory/Exists.cs

```diff
@@ -53,17 +53,14 @@ public void ValidPathExists_ReturnsTrue()
             });
         }
 
-        [Fact]
-        public void PathWithInvalidCharactersAsPath_ReturnsFalse()
+        [Theory MemberData("PathsWithInvalidCharacters")]
```

Comment:
It's the intended syntax of the language.  This was a bad omission in Roslyn 1.0 that we only recently caught.  The full impact, considering future features, hasn't ever been done for this.  Hence we moved back to the intended syntax.